spec/features/visitors/sign_up_spec.rb

```diff
@@ -209,12 +213,12 @@
     expect(current_url).to eq confirm_url
   end
 
-  scenario 'visitor confirms more than once' do
+  scenario 'visitor confirms more than once while signed in' do
```

Comment:
Are we not still expecting to see a flash here?

Perhaps the name of this test could be better... something describes what we expect to happen?

`scenario 'signed in users who re-visit the confirmation page are redirected to 2FA setup'`